### DISCUSSION TITLE
After closing the window, displaying the window again will not respond

### DISCUSSION THREAD

**Discussion by cplasfwst** [fyne-io/fyne#4724] [Q&A]
My program has a total of 2 windows, and my second window is intended to display the program's running logs. The first log window was opened normally, but when I manually closed the window, clicking the button did not display the window again
Here are the relevant codes:
`//关于查看任务
	form := widget.NewForm()
	myWindow2 := myApp.NewWindow(`查看任务情况`)
	myWindow2.SetContent(container.NewVScroll(form))`

`//查看运行情况
	chakanrenwu.OnTapped = func() {
		myWindow2.Show()

		//chaolianjie := widget.NewRichTextFromMarkdown(``)
		//chaolianjie2 := widget.NewRichTextFromMarkdown(``)

		//chaolianjie.Segments = ``

		//form.Append(`第一`, chaolianjie)
		//form.Append(`第而`, chaolianjie2)
		item := widget.NewFormItem("第"+strconv.Itoa(1)+`次任务`, widget.NewRichTextFromMarkdown(`[点击查看:](`+`url`+`)`))
		form.AppendItem(item)
	}`

In the actual operation of the program, I closed myWindows 2 and the second time I clicked open, I couldn't jump out of the window

**Comment by andydotxyz** [comment]
Calling `Window.Close` will destroy it. If you want to re-show a window then use `Hide` instead.
